cluster-scope/overlays/nerc-ocp-test-2/nodenetworkconfigurationpolicies/vlan-211.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: vlan-211
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "wrk-5"
+  desiredState:
+    interfaces:
+    - name: eno1np0.211
+      type: vlan
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+        auto-routes: true
+        address:
+        - ip: 10.0.120.39
+          prefix-length: 22
+      vlan:
+        base-iface: eno1np0
+        id: 211
+      mtu: 9000
+    routes:
+      config:
```

Comment:
Are you certain we need these static routes? This information should be provided by DHCP.

Reply:
"I don't think there's any dhcp for tagged networks" - [Mainn](https://massopencloud.slack.com/archives/CG1C4SZCM/p1723046737555839?thread_ts=1722974917.195019&cid=CG1C4SZCM) 
When the vlan interface was created it did not get assigned an ip, there's a discussion in the linked thread about it. Hakan said to set that static IP.

Reply:
We should be able to use DHCP. Tagged/not tagged doesn't have any impact on whether or not there is DHCP available.